endpoints/users.js

```diff
@@ -5,24 +5,40 @@ const config = require('../config');
 
 create = (request, response) => {
     if (!request.body.username) {
-        response.status(400).json({"err": "Missing username field."});
-    }
-    if (!request.body.password) {
-        response.status(400).json({"err": "Missing password field."});
-    }
+        response.status(400).json({"err": "Missing username field."}).end();
+    } else if (!request.body.password) {
+        response.status(400).json({"err": "Missing password field."}).end();
+    } else if (request.body.username.length < 5){
+        response.status(400).json({"err": "Username must be 5 or more characters."}).end();
```

Comment:
This error should probably be in the same if clause as the password errors and simply add it into the JSON array of errors.